addons/sky_3d/src/TimeOfDay.gd

```diff
@@ -130,28 +138,28 @@ var year: int = 2025: set = set_year
 func set_total_hours(value: float) -> void:
 	total_hours = value
 	emit_signal("time_changed", value)
-	if Engine.is_editor_hint():
+	if is_editor_hint_or_manual():
 		__set_celestial_coords()
 
 
 func set_day(value: int) -> void:
 	day = value 
 	emit_signal("day_changed", value)
-	if Engine.is_editor_hint():
+	if is_editor_hint_or_manual():
 		__set_celestial_coords()
```

Comment:
Change to:
```
func set_day(value: int) -> void:
	if day != value:
		day = value 
		emit_signal("day_changed", value)
	update()
```

Change all the rest similarly. The check is only needed if emitting signals. They can all always update.